cms/templates/widgets/header.html

```diff
@@ -98,6 +99,9 @@ <h3 class="title"><span class="label"><span class="label-prefix sr">${_("Course"
                   <li class="nav-item nav-course-settings-advanced">
                     <a href="${advanced_settings_url}">${_("Advanced Settings")}</a>
                   </li>
+                  <li class="nav-item nav-course-settings-certificates">
```

Comment:
@asadiqbal08 -- we need to show this link only if the certificates web view has been enabled in configuration -- can you add a check for the "CERTIFICATES_HTML_VIEW" Django setting?